src/bin/pgcopydb/copydb.h

```diff
@@ -76,7 +76,7 @@ typedef struct TransactionSnapshot
 	TransactionSnapshotKind kind;
 	TransactionSnapshotState state;
 
-	char pguri[MAXCONNINFO];
+	char *pguri;  	/* malloc'ed area */
```

Comment:
Is it really though? I believe now it just points to a memory area that's allocated/managed elsewhere, right?

Reply:
No, actually it is allocated memory in `copydb_copy_snapshot` Line 32